tests/unit/test_alias.py

```diff
@@ -0,0 +1,432 @@
+            alias_interface.get_aliases(),
+            {'my-alias': 'my-alias-value --my-parameter my-value'})
+
+    def test_get_aliases_with_alias_that_includes_newlines(self):
+        with open(self.alias_file, 'a+') as f:
+            f.write('my-alias = my-alias-value\n')
+        alias_interface = AliasLoader(self.alias_file)
+        self.assertEqual(
+            alias_interface.get_aliases(), {'my-alias': 'my-alias-value'})
+
+    def test_get_aliases_with_alias_that_is_indented(self):
+        with open(self.alias_file, 'a+') as f:
+            f.write('my-alias = \n  my-alias-value\n')
+        alias_interface = AliasLoader(self.alias_file)
+        self.assertEqual(
+            alias_interface.get_aliases(), {'my-alias': 'my-alias-value'})
+
+    def test_get_aliases_with_multiple_lines(self):
+        with open(self.alias_file, 'a+') as f:
+            f.write(
+                'my-alias = \n'
+                '  my-alias-value \ \n'
+                '  --parameter foo\n')
+        alias_interface = AliasLoader(self.alias_file)
+        self.assertEqual(
+            alias_interface.get_aliases(),
+            # The backslash and newline should be preserved, but
+            # still have the beginning and end newlines removed.
+            {'my-alias': 'my-alias-value \\\n--parameter foo'})
+
+    def test_get_aliases_with_multiple_aliases(self):
+        with open(self.alias_file, 'a+') as f:
+            f.write('my-alias = my-alias-value\n')
+            f.write('my-second-alias = my-second-alias-value\n')
+        alias_interface = AliasLoader(self.alias_file)
+        self.assertEqual(
+            alias_interface.get_aliases(),
+            {'my-alias': 'my-alias-value',
+             'my-second-alias': 'my-second-alias-value'})
+
+
+class TestBaseAliasCommand(unittest.TestCase):
+    def test_name(self):
+        alias_cmd = BaseAliasCommand('alias-name', 'alias-value')
+        self.assertEqual(alias_cmd.name, 'alias-name')
+        alias_cmd.name = 'new-alias-name'
+        self.assertEqual(alias_cmd.name, 'new-alias-name')
+
+
+class TestServiceAliasCommand(unittest.TestCase):
+    def setUp(self):
+        self.alias_name = 'myalias'
+        self.session = mock.Mock()
+
+    def create_command_table(self, services):
+        command_table = {}
+        for service in services:
+            command_table[service] = mock.Mock()
+        return command_table
+
+    def create_parser(self, command_table, extra_params=None):
+        parser = MainArgParser(
+            command_table=command_table,
+            version_string='version',
+            description='description',
+            argument_table={}
+        )
+        if extra_params:
+            for extra_param in extra_params:
+                parser.add_argument('--'+extra_param)
+        return parser
+
+    def test_alias_with_only_service_command(self):
+        alias_value = 'myservice'
+
+        command_table = self.create_command_table(['myservice'])
```

Comment:
It might be helpful to show how the values propagate here (use `alias_value` instead of the hard coded string).